src/ripple/protocol/impl/STObject.cpp

```diff
@@ -76,7 +76,7 @@ STObject::STObject (SerialIter& sit, SField const& name, int depth)
 }
 
 STObject&
-STObject::operator= (STObject&& other)
+STObject::operator= (STObject&& other) noexcept
 {
     setFName(other.getFName());
```

Comment:
`setFName(other.getFName())` is not noexcept.

Reply:
Fixed in 68c3ec29f1be6088dba866e6685752cf40507b25